app/models/pet.rb

```diff
@@ -14,6 +14,6 @@ def self.adoptable
   end
 
   def self.search(name)
-    where(name: name)
+    where("name ILIKE ?", "%#{name}%")
```

Comment:
I want to say you can just do `where("name ILIKE ?", name)`? It should just plug name straight into the question mark!